day_7/index.js

```diff
@@ -1,320 +1,543 @@
-
-    }
-
-    return console.log(array);
-}
-
-const generateColors = (numberName, count) => {
-    if (numberName == 'hex') {
-        arrayOfHexaColors2(count);
-    } else if (numberName == 'rgb') {
-        rgbColorGenerator(count);
-
-    }
-
-}
-
-generateColors('rgb', 2);
\ No newline at end of file
+const fullName = (firstName = 'Артур', lastName = 'Москвичёв') => {
+  let space = ' ';
+  let fullName = firstName + space + lastName;
+  return console.log(fullName);
+};
+fullName('Денис', 'Токарев');
+
+const addNumbers = (firstNumber, secondNumber) => {
+  let sum = firstNumber + secondNumber;
+  return console.log(sum);
+};
+
+addNumbers(1, 2);
+
+const areaOfRectangle = (length, width) => {
+  let area = length * width;
+  return console.log(area);
+};
+
+areaOfRectangle(10, 8);
+
+const perimeterOfRectangle = (length, width) => {
+  let perimeter = 2 * (length + width);
+  return console.log(perimeter);
+};
+
+perimeterOfRectangle(10, 8);
+
+const volumeOfRectPrism = (length, width, heigth) => {
+  let volume = length * width * heigth;
+  return console.log(volume);
+};
+
+volumeOfRectPrism(10, 12, 13);
+
+const areaOfCircle = (radius) => {
+  let areaCircle = Math.PI * radius * radius;
+  return console.log(areaCircle);
+};
+
+areaOfCircle(10);
+
+const lengthOfCircle = (radius) => {
+  let lengthCircle = 2 * Math.PI * radius;
+  return console.log(lengthCircle);
+};
+
+lengthOfCircle(10);
+
+const density = (weight, volume) => {
+  let Density = weight / volume;
+  return console.log(Density);
+};
+
+density(10, 12);
+
+const speed = (distance, time) => {
+  let speed = distance / time;
+  return console.log(speed);
+};
+
+speed(100, 2);
+
+const ITM = (weight, heigth) => {
+  let itm = weight / (heigth * heigth);
+  if (itm < 18.5) {
+    return console.log('Недостаточный вес');
+  } else if (itm >= 18.5 && itm <= 24.9) {
+    return console.log('Нормальный вес');
+  } else if (itm >= 25 && itm <= 29.9) {
+    return console.log('Избыточный вес');
+  } else {
+    return console.log('Ожирение');
+  }
+};
+
+ITM(180, 108);
+
+const checkSeason = (month) => {
+  if (month == 'декабрь' || month == 'январь' || month == 'февраль') {
+    return console.log('зима');
+  } else if (month == 'март' || month == 'апрель' || month == 'май') {
+    return console.log('весна');
+  } else if (month == 'июль' || month == 'июнь' || month == 'август') {
+    return console.log('лето');
+  } else if (month == 'сентябрь' || month == 'октяюбрь' || month == 'ноябрь') {
+    return console.log('осень');
+  }
+};
+
+checkSeason('февраль');
+
+const findMax = (firstNumber, secondNumber, thirdNumber) => {
+  if (firstNumber >= secondNumber && firstNumber >= thirdNumber) {
+    return console.log(firstNumber);
+  } else if (secondNumber >= firstNumber && secondNumber >= thirdNumber) {
+    return console.log(secondNumber);
+  } else if (thirdNumber >= firstNumber && thirdNumber >= secondNumber) {
+    return console.log(thirdNumber);
+  }
+};
```

Comment:
Опять-таки, можно упростить как синтаксически, так и логически:
```suggestion
const findMax = (firstNumber, secondNumber, thirdNumber) => {
  if (firstNumber >= secondNumber && firstNumber >= thirdNumber) {
    // Здесь firstNumber больше, чем и secondNumber, и thirdNumber
    return console.log(firstNumber);
  } 
  
  // Здесь firstNumber меньше либо secondNumber, либо thirdNumber
  // Это значит, что firstNumber точно не максимум, и максимум надо искать среди
  // secondNumber и thirdNumber, даже если одно из этих чисел может быть меньше,
  // чем firstNumber – нам уже похуй на firstNumber в принципе, потому что он 
  // заведомо не подходит
  if (secondNumber >= thirdNumber) {
    // Раз secondNumber больше thirdNumber, он самый большой 
    // (а thirdNumber либо самый маленький, либо средний, но нам это всё уже не важно)
    return console.log(secondNumber);
  } 
    
  // Если secondNumber меньше thirdNumber, то thirdNumber самый большой
  return console.log(thirdNumber);
};
```